spec/include/ItemHeatingCooling.vspec

```diff
@@ -0,0 +1,14 @@
+#
+# All files and artifacts in this repository are licensed under the
+# provisions of the license provided by the LICENSE file in this repository.
+#
+
+
+HeatingCooling:
```

Comment:
Changed `TemperatureControl` (in previous PR version) to `HeatingCooling`, I think that I prefer that over `RelativeTemperature` but we can discuss that at next meeting